Example/App.tsx

```diff
@@ -0,0 +1,168 @@
+import BottomTabsAndStack from './src/screens/BottomTabsAndStack';
+import StackReactNavigation4 from './src/screens/StackReactNavigation4';
+import Modals from './src/screens/Modals';
+import Orientation from './src/screens/Orientation';
+
+enableScreens();
+
+const SCREENS: Record<
+  string,
+  {
+    title: string;
+    component: () => JSX.Element;
+    type: 'example' | 'playground';
+  }
+> = {
+  SimpleNativeStack: {
+    title: 'Simple Native Stack',
+    component: SimpleNativeStack,
+    type: 'example',
+  },
+  StackPresentation: {
+    title: 'Stack Presentation',
+    component: StackPresentation,
+    type: 'example',
+  },
+  BottomTabsAndStack: {
+    title: 'Bottom tabs and native stack',
+    component: BottomTabsAndStack,
+    type: 'example',
+  },
+  Modals: {
+    title: 'Modals',
+    component: Modals,
+    type: 'example',
+  },
+  StackReactNavigation4: {
+    title: 'Stack react-navigation v4',
+    // @ts-ignore react-navigation v4 AppNavigator type
+    component: StackReactNavigation4,
+    type: 'example',
+  },
+  HeaderOptions: {
+    title: 'Header Options',
+    component: HeaderOptions,
+    type: 'playground',
+  },
+  StatusBar: {
+    title: 'Status bar (iOS)',
+    component: StatusBar,
+    type: 'playground',
+  },
+  Animations: {
+    title: 'Animations',
+    component: Animations,
+    type: 'playground',
+  },
+  Orientation: {
+    title: 'Orientation',
+    component: Orientation,
+    type: 'playground',
+  },
+};
+
+type RootStackParamList = {
+  Main: undefined;
+} & {
+  [P in keyof typeof SCREENS]: undefined;
+};
+
+const Stack = createNativeStackNavigator<RootStackParamList>();
+
+interface MainScreenProps {
+  navigation: StackNavigationProp<RootStackParamList, 'Main'>;
+}
+
+const MainScreen = ({navigation}: MainScreenProps): JSX.Element => (
+  <ScrollView>
+    <SafeAreaView>
+      <SettingsSwitch
+        style={styles.switch}
+        label="Right to left"
+        value={I18nManager.isRTL}
+        onValueChange={() => {
+          I18nManager.forceRTL(!I18nManager.isRTL);
+          RNRestart.Restart();
+        }}
+      />
+      <Text style={styles.label}>Examples</Text>
+      {Object.keys(SCREENS)
+        .filter((name) => SCREENS[name].type === 'example')
+        .map((name) => (
+          <MenuItem
+            key={name}
+            title={SCREENS[name].title}
+            onPress={() => navigation.navigate(name)}
+          />
+        ))}
+      <Text style={styles.label}>Playgrounds</Text>
+      {Object.keys(SCREENS)
+        .filter((name) => SCREENS[name].type === 'playground')
+        .map((name) => (
+          <MenuItem
+            key={name}
+            title={SCREENS[name].title}
+            onPress={() => navigation.navigate(name)}
+          />
+        ))}
+    </SafeAreaView>
+  </ScrollView>
+);
+
+const ExampleApp = (): JSX.Element => (
+  <NavigationContainer>
+    <Stack.Navigator
+      screenOptions={{
+        statusBarStyle: 'dark',
```

Comment:
Shouldn't we apply `rtl`/`ltr` here too?

Reply:
Added in https://github.com/software-mansion/react-native-screens/pull/827/commits/85e8e17b756652507b71956a30319822422c5358